services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/organisaatio/OrganisaatioControllerTests.kt

```diff
@@ -0,0 +1,64 @@
+package fi.hel.haitaton.hanke.organisaatio
+
+import com.ninjasquad.springmockk.MockkBean
+import fi.hel.haitaton.hanke.HankeService
+import io.mockk.every
+import org.hamcrest.Matchers.containsInAnyOrder
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.*
+
+/**
+ * For testing Spring Boot Actuator endpoints
+ */
+@SpringBootTest(
+    properties = [
+        "management.server.port=",
+        "spring.liquibase.enabled=false",
+        "spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration"
+    ]
+)
+@AutoConfigureMockMvc
+@EnableAutoConfiguration
+class OrgansaatioControllerEndpointTests(@Autowired val mockMvc: MockMvc) {
+
+    // Just to prevent the context trying to init that service, and fail doing it.
+    @MockBean
+    lateinit var hankeService: HankeService
+
+    @MockkBean
+    lateinit var organisaatioService: OrganisaatioService
+
```

Comment:
Same thing: Either use MockBean or MockkBean. With Hanke tests there are now Mockk in integration tests and Mockito in unit tests but only one in each test class.

Mockito is more familiar to me so I tend to use that but Teemu got integration test behave correctly with Mockk which is his favorite.

Reply:
I think the changes coming from Teemu's PR will make the other (HankeService) mocking obsolete, so this issue will get resolved without changes needed here.